README.md

```diff
@@ -1,103 +1,89 @@
 *   app:cpb_foregroundProgressWidth  (Integer)  ->  default 10
 *   app:cpb_backgroundProgressWidth  (Integer)  ->  default 10
-*   app:cpb_backgroundProgressColor  (Color)    ->  default Color.GRAY
-*   app:cpb_foregroundProgressColor  (Color)    ->  default Color.BLACK
+*   app:cpb_backgroundProgressColor  (Color)    ->  default ohos.agp.utils.Color.GRAY.getValue()
+*   app:cpb_foregroundProgressColor  (Color)    ->  default ohos.agp.utils.Color.BLACK.getValue()
 *   app:cpb_progress                 (Float)    ->  default 0
 *   app:cpb_touchEnabled             (boolean)  ->  default false
 *   app:cpb_clockwise                (boolean)  ->  default false
 
-JAVA
------
-
+#### Java:
 ```java
-CircleProgressbar circleProgressbar = (CircleProgressbar)findViewById(R.id.yourCircularProgressbar);
-circleProgressbar.setForegroundProgressColor(Color.RED);
-circleProgressbar.setBackgroundColor(Color.GREEN);
-circleProgressbar.setBackgroundProgressWidth(15);
-circleProgressbar.setForegroundProgressWidth(20);
-circleProgressbar.enabledTouch(true);
-circleProgressbar.setRoundedCorner(true);
-circleProgressbar.setClockwise(true);
-int animationDuration = 2500; // 2500ms = 2,5s
-circleProgressbar.setProgressWithAnimation(65, animationDuration); // Default duration = 1500ms
+    CircleProgressbar cpb = (CircleProgressbar) findComponentById(ResourceTable.Id_circleprogressbar);
+    cpb.setForegroundProgressColor(Color.RED.getValue());
+    cpb.setBackgroundProgressColor(Color.GREEN.getValue());
+    cpb.setBackgroundProgressWidth(15);
+    cpb.setForegroundProgressWidth(20);
+    cpb.enabledTouch(true);
+    cpb.setRoundedCorner(true);
+    cpb.setClockwise(true);
 ```
-* There is progress listener interface you can implement as par your requierments and there are also getter for all the above methods.
-
-## How to contribute?
-
-1. Fork the repository 
-2. Do the desired changes (add/delete/modify)
-3. Make a pull request
 
-## When to contribute?
-
-1. Fix open bugs.
-2. Add new issue.
-
-
-LICENCE
------
-
- Copyright 2017 Ankit kumar
-
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
-
- Thanks to stackoverflow and Raggav
+## Future Work
+Due to unavailability of similar api as ObjectAnimator in HarmonyOS platform, the animation behaviour is not exactly the same as android base library but it is similar.
+                                                                                                   
+## Licence
+```
```

Comment:
Keep the license section same as the base library.

Reply:
Done